barretenberg/cpp/src/barretenberg/ultra_honk/decider_proving_key.hpp

```diff
@@ -200,11 +200,14 @@ template <IsUltraOrMegaHonk Flavor> class DeciderProvingKey_ {
                 public_inputs.emplace_back(polynomials.w_r[idx]);
             }
 
-            // Set the pairing point accumulator indices. This should exist for all flavors.
-            ASSERT(circuit.pairing_inputs_public_input_key.is_set() &&
-                   "Honk circuit must output a pairing point accumulator. If this is a test, you might need to add a \
+            // Set the pairing point accumulator indices. This should exist for all non-Mega.
```

Comment:
Just updating some debug asserts on `pairing_inputs_public_input_key` to exclude Mega since it is no longer used there